pybamm/input/parameters/lithium-ion/lithium_platings/yang2017_Li_plating/plating_exchange_current_density_OKane2020.py

```diff
@@ -0,0 +1,26 @@
+from pybamm import Scalar
+
+
+def plating_exchange_current_density_OKane2020(c_e, c_Li, T):
```

Comment:
is this really the function in OKane? I thought that had a c_e or c_Li dependence?

Reply:
I think that is the one from Yang et al. If so, it should be renamed.

Reply:
still needs to be renamed (it's in the right folder but misleading name)